kubeflow/asm/Makefile

```diff
@@ -1,11 +1,17 @@
-# Get a list of stable versions of install_asm and config by running this command
-# curl https://storage.googleapis.com/csm-artifacts/asm/STABLE_VERSIONS
-# For example: 1.9.2-asm.1+config4:install_asm_1.9.2-asm.1-config4
+# ASM was upgraded to v1.13 in Kubeflow for GCP v1.5.1
+# ASM changed the installation process begining v1.12. See details:
+# https://cloud.google.com/service-mesh/docs/unified-install/anthos-service-mesh-prerequisites
+# To upgrade your current ASM instance, please follow instructions:
+# https://cloud.google.com/service-mesh/docs/unified-install/plan-upgrade
+#
+# Get a list of stable versions of asmcli and config by running this command
+# curl https://storage.googleapis.com/csm-artifacts/asm/ASMCLI_VERSIONS
+# For example: 1.13.4-asm.4+config2:asmcli
 # The part before colon symbol should be used for ASM_PACKAGE_VERSION.
-# The part after colon symbol should be used for INSTALL_ASM_SCRIPT_VERSION.
-ASM_PACKAGE_VERSION=1.10.4-asm.6+config2
-INSTALL_ASM_SCRIPT_VERSION=install_asm_1.10.4-asm.6-config2
-# ASM Upgrade Guide: Check out README.md of the same directory.
+# The part after colon symbol should be used for ASMCLI_SCRIPT_VERSION.
+
+ASM_PACKAGE_VERSION=1.13.4-asm.4+config2
+ASMCLI_SCRIPT_VERSION=asmcli
```

Comment:
This value `asmcli` is a pointer, which can dynamically change to the latest released asmcli. My suggestion is to use the exact version of `asmcli` tool like `asmcli_1.13.4-asm.4-config2` which guarantees the stability of kubeflow deployment.

Reply:
If `asmcli_1.13.4-asm.4-config2` as asmcli is failing, we can try this combination: `1.13.2-asm.5+config2:asmcli_1.13.2-asm.5-config2`

Reply:
Done